docs/sphinx/source/conf.py

```diff
@@ -86,6 +87,15 @@
 
 master_doc = 'index'
 
+# Example configuration for intersphinx: refer to the Python standard library.
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3.8/', None),
+    'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
+    'numpy': ('https://numpy.org/doc/stable/', None),
+    'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
+    'pvlib': ('https://pvlib-python.readthedocs.io/en/stable/', None),
```

Comment:
I think it's worth pausing and thinking about what versions make the most sense to link against -- `stable` seems like the right choice to me for packages, but what about python itself?  Link against the version we expect people to be using (I'm guessing 3.7 or 3.8?), or maybe the most recent stable version, so 3.9?

Reply:
Sorry it should just say "3" eg: https://docs.python.org/3/ not 3.8. Python 3 is stable and always points to the most recent version. But you're right, that's 3.9. Hmm, up to you all I guess